src/xamarin-client/Adaptive.ReactiveTrader.Client.Domain/Transport/Wamp/ServiceObservableExtensions.cs

```diff
@@ -93,7 +93,7 @@ public static class ServiceObservableExtensions
                 disposables.Add(
                     source.Subscribe(innerSource =>
                     {
-                        IObservable<TValue> innerSourcePublished = innerSource.Replay(1).RefCount();
+                        IObservable<TValue> innerSourcePublished = innerSource.Publish().RefCount();
```

Comment:
Think you should roll back, it changes the semantics. It was probably a change I made in the middle of the night to try and get the thing working.